Description: Sphinx 1.4+ has new values
 Newer Sphix releases populated indexentries with one element. Its common
 value is 'None' - it's get ignored here. entries also have one more, also
 ignored.
Author: Laszlo Boszormenyi (GCS) <gcs@debian.org>
Last-Update: 2016-07-16

---

--- python-gevent-1.1.1.orig/doc/mysphinxext.py
+++ python-gevent-1.1.1/doc/mysphinxext.py
@@ -46,9 +46,9 @@ def missing_reference(app, env, node, co
     for docname, items in env.indexentries.items():
         if noisy >= 2:
             print(docname)
-        for (i_type, i_string, i_target, i_aliasname) in items:
+        for (i_type, i_string, i_target, i_aliasname, i_ignored) in items:
             if noisy >= 3:
-                print('---', [i_type, i_string, i_target, i_aliasname])
+                print('---', [i_type, i_string, i_target, i_aliasname, i_ignored])
             if i_aliasname.endswith(target):
                 stripped_aliasname = i_aliasname[len(docname):]
                 if stripped_aliasname:
--- python-gevent-1.1.1.orig/doc/mytheme/genindex.html
+++ python-gevent-1.1.1/doc/mytheme/genindex.html
@@ -17,7 +17,7 @@
 {%- set breakat = genindexcounts[loop.index0] // 2 %}
 {%- set numcols = 1 %}
 {%- set numitems = 0 %}
-{% for entryname, (links, subitems) in entries %}
+{% for entryname, (links, subitems, ignored) in entries %}
 <dt>{%- if links -%}<a href="{{ links[0] }}">{{ entryname|e }}</a>
   {%- for link in links[1:] %}, <a href="{{ link }}">[{{ loop.index }}]</a>{% endfor -%}
   {%- else -%}
